.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "**/*.{mjs,ts,tsx,md,mdx}": ["eslint --fix"],
+  "**/*.{mjs,ts,tsx,md,mdx,json,yml}": ["prettier --write"]
```

Comment:
I am curious to see if this will be sequential or in parallel. I know we had problems in CI with these too tools competing.

Reply:
They run in serial, afaics